polls/tool/src/webapp/templates/voteAdd.html

```diff
@@ -37,7 +37,9 @@
 
 <body rsf:id="scr=sakai-body">
 <div class="portletBody container-fluid">
-<h3 rsf:id="new_poll_title">Add/Edit a poll</h3>
+	<div class="page-header">
```

Comment:
This file seems to use spaces instead of tabs for the indenting. Please match the neighbouring indenting. Thanks.